cpp/lazperf/vlr.hpp

```diff
@@ -159,6 +159,67 @@ struct LAZPERF_EXPORT wkt_vlr : public vlr
     virtual vlr_header header() const;
 };
 
+
+struct LAZPERF_EXPORT copc_extents_vlr : public lazperf::vlr
+{
+public:
+
+    struct LAZPERF_EXPORT CopcExtent
+    {
+        double minimum {0.0};
+        double maximum {0.0};
```

Comment:
What do we think about naming them min/max? Easier to write no?
```suggestion
        double min {0.0};
        double max {0.0};
```

Reply:
min and max clash with windows macro names that are commonly defined. Purposely chose these so we never have to worry about `#ifdef NOMINMAX`